api/src/prescription/campaign/infrastructure/repositories/campaign-assessment-participation-result-list-repository.js

```diff
@@ -155,7 +154,7 @@ function _filterByStage(queryBuilder, stageCollection, filters) {
 }
 
 async function _buildCampaignAssessmentParticipationResultList(results, stageCollection) {
-  return await bluebird.mapSeries(results, async (result) => {
+  return PromiseUtils.mapSeries(results, async (result) => {
```

Comment:
pourquoi ne pas avoir transformer en boucle `for( const result of results)` ici ?

Reply:
Car le retour du map est utilisé 😉
```
const participations = await _buildCampaignAssessmentParticipationResultList(results, stageCollection);
return {
    participations,
    pagination,
  };
```